docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
       - ./packages/discord-bot:/usr/src/app/packages/discord-bot
       - ./package.json:/usr/src/app/package.json
       - ./tsconfig.base.json:/usr/src/app/tsconfig.base.json
+      - ./schema.graphql:/usr/src/app/schema.graphql
```

Comment:
Is this so the container doesn't have to generate this itself?

Reply:
The problem was that when the schema was updated, we had to rebuild the backend docker image. The watchers that runs inside can't know about changes in hasura so doesn't regenerate it (as opposed to TS files generated from this schema and queries in our code)